plugins/custom-webpack-plugin/index.js

```diff
@@ -26,6 +26,7 @@ module.exports = function (context, options) {
             http: require.resolve('stream-http'),
             https: require.resolve('https-browserify'),
             stream: require.resolve('stream-browserify'),
+            os: require.resolve('os-browserify/browser'),
```

Comment:
why using this?

Reply:
During development, I had a polyfill error related to this. It specifically needed this package for some reason and when I added it, it worked fine